install/install.py

```diff
@@ -38,8 +38,8 @@ def create_directory(self, directory):
             return False
 
     def run_command(self, command):
-        print("[*] Running following command")
-        print("%s"%(command))
+        Colorizer.normal("[*] Running following command")
+        Colorizer.info("%s"%(command))
```

Comment:
So, one white space to the left, and right of %, plus removal of redundant parenthesis around `command`, right ?